server/app/services/export/CsvExporter.java

```diff
@@ -58,11 +71,58 @@ public void exportRecord(
       ReadOnlyApplicantProgramService roApplicantService,
       Optional<Boolean> optionalEligibilityStatus)
       throws IOException {
-    ImmutableMap<Path, String> answerMap =
-        roApplicantService.getSummaryData().stream()
-            .flatMap(data -> data.scalarAnswersInDefaultLocale().entrySet().stream())
-            .collect(ImmutableMap.toImmutableMap(Map.Entry::getKey, Map.Entry::getValue));
+    Map<Path, String> answerDataMap = new HashMap<>();
```

Comment:
**suggestion**: I think we can use an `ImmutableMapBuilder` here instead. Then we won't have to convert to the immutable `AnswerMap` at the end, and can leave this named `answerMap` (since it's a map of answers, not of `AnswerData`s.)

Reply:
Done